cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/s2i-python/s2i-python-task.yaml

```diff
@@ -43,6 +43,10 @@ spec:
       type: array
       description: Environment variables to set during _build-time_.
       default: []
+    - name: BASE_IMAGE
+      type: string
+      description: The location of the base python image.
+      default: []
```

Comment:
@sveno1990 thanks for the PR, can you please update the `default` as `image-registry.openshift-image-registry.svc:5000/openshift/python` ?

Reply:
Thanks @jkandasa, my bad, issue should now be fixed.